### DISCUSSION TITLE
Unable to set session id or other CallbackHandler parameters when passing trace client in langchain sdk

### DISCUSSION THREAD

**Discussion by sasidevtool** [langfuse/langfuse#1184] [Support] [answered]
### Describe the bug

When stateful_client parameter in CallbackHandler is set rest of the parameters like session_id are ignored and not set.

### To reproduce

langfuse = Langfuse()
trace_client = langfuse.trace(name=trace_name, tags=[my tags])
callbackhandler = CallbackHandler(session_id=session_id, stateful_client= trace_client)

### Additional information

There are two ways this can be fixed

1. Allow rest of the parameters to be set when CallbackHandler is initialised with stateful_client
2. Or provide session id and other parameters in langfuse.trace similar to python sdk

**Comment by marcklingen** [comment, marked as answer]
If you want to use the Langfuse Python SDK and Langchain integration, do the following:
```python
langfuse = Langfuse()
trace = langfuse.trace(name=trace_name, tags=["my tags"], session_id=session_id)
langchain_callback_handler = trace.get_langchain_handler()
```

Docs: https://langfuse.com/docs/integrations/langchain/python#adding-trace-as-context-to-a-langchain-handler

**Reply by sasidevtool** [reply]
Thank you @marcklingen . This worked.
